src/main/java/com/mineinabyss/looty/ecs/action/DecreaseDurabilityAction.kt

```diff
@@ -0,0 +1,17 @@
+package com.mineinabyss.looty.ecs.action
+
+import com.mineinabyss.geary.ecs.api.actions.GearyAction
+import com.mineinabyss.geary.ecs.api.entities.GearyEntity
+import com.mineinabyss.looty.ecs.components.DurabilityComponent
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("looty:decreaseDurability")
+class DecreaseDurabilityAction(private val deltaDurability: Int) : GearyAction() {
```

Comment:
Feels like these could be merged into one action where the delta can be either positive or negative.